File: internal/content/testdata/bench/go/best-time-to-buy-stock.go
func max_profit(prices []int) int {
	minPrice := prices[0]
	profit := 0
	for _, p := range prices[1:] {
		if p-minPrice > profit {
			profit = p - minPrice
		}
		if p < minPrice {
			minPrice = p
		}
	}
	return profit
}

File: internal/content/testdata/bench/rust/best-time-to-buy-stock.rs
fn max_profit(prices: Vec<i64>) -> i64 {
    let mut min_price = prices[0];
    let mut profit: i64 = 0;
    for &p in &prices[1..] {
        profit = profit.max(p - min_price);
        min_price = min_price.min(p);
    }
    profit
}
